### PR TITLE
Added Responsiveness To Experience Page

### PR DESCRIPTION
**Pull request by Atif0604**
Hey @smaranjitghose
This PR fixes issue Fix the responsiveness issues for the Expereince Page in Mobile Screens #801
Please Review my PR

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -24,78 +24,6 @@ let header = $(`
    <li class="nav-item">
    <input type="checkbox" class="dark_toggler" aria-label="Toggle Light Mode" onclick="toggle_light_mode()">
    </li>
-   <div class="bike">
-   <svg xmlns="http://www.w3.org/2000/svg" viewBox="-80 0 650 400" preserveAspectRatio="xMinYMin meet">
-     <rect/>
-     <g>
-       <g id="man-bike">
-         <path id="Vector_2" opacity="0.1" d="M201.091 276.806L179.893 255.76L173.269 262.513L194.467 283.559L201.091 276.806Z" fill="black" />
```
Comment (smaranjitghose) [inline assets/js/app.js:32]: Why is this removed?

File: experience.html [hunk 2]
```diff
@@ -54,8 +54,9 @@
             <div class="electron-omega"></div>
 		</div>
 
-    </div>tps://dribbble.com/assets/logo-small-2x-9fe74d2ad7b25fba0f50168523c15fda4c35534f9ea0b1011179275383035439.png" /></a>
     </div>
+    <!-- <a href = "https://dribbble.com/assets/logo-small-2x-9fe74d2ad7b25fba0f50168523c15fda4c35534f9ea0b1011179275383035439.png" /></a>
```
Comment (smaranjitghose) [inline experience.html:58]: Not used anymore!

File: assets/js/app.js (hunk 1 above)
Comment (Atif0604) [reply assets/js/app.js:32]: because it was making the navbar long which made white space in this sides and also along the page and horizontal scroll ,also this element  wasn't used anywhere

**Comment by smaranjitghose** [comment]
## Multiple Responsiveness Issues still persist

Mob_15
------------------------------------------------
-------------------------------------------------
Mob_14
------------------------------------------------
-------------------------------------------------
Mob_13
------------------------------------------------
-------------------------------------------------
Mob_12
------------------------------------------------
-------------------------------------------------
Mob_11

**Comment by Atif0604** [comment]
> ## Multiple Responsiveness Issues still persist
> ## Mob_15
> ## Mob_14
> ## Mob_13
> ## Mob_12
> Mob_11

ohk i will work on this @smaranjitghose

File: assets/js/app.js (hunk 1 above)
Comment (smaranjitghose) [reply assets/js/app.js:32]: This element is used in the sidebar. Please have a look at the mobile screen sizes using Chrome Tools.

Comment (Atif0604) [reply assets/js/app.js:32]: ok i will add it

**Comment by Atif0604** [comment]
@smaranjitghose made the changes

File: assets/css/experience.css [hunk 3]
```diff
@@ -526,24 +531,26 @@ ul ol{
...
+ /* .content > * + * {
   margin-top: 1rem;
-} 
+}  */
 
 .volunteerTitle {
   font-size: 1.5rem;
   line-height: 1.2;
-  margin-top: 0.5em;
+  /* margin-top: 0.5em; */
```
Comment (smaranjitghose) [inline assets/css/experience.css:541]: Delete the unused code please

**Review by smaranjitghose — APPROVED** [review]

File: assets/css/experience.css (hunk 3 above)
Comment (smaranjitghose) [reply assets/css/experience.css:541]: Remove all the commented out code. The PR is ready for merge after that

Comment (Atif0604) [reply assets/css/experience.css:541]: remove all the commented codes or the code which i commented? @smaranjitghose

Comment (smaranjitghose) [reply assets/css/experience.css:541]: All the commented codes please

Comment (Atif0604) [reply assets/css/experience.css]: Done @smaranjitghose

**Comment by Atif0604** [comment]
@smaranjitghose when it will be merged?

**Comment by smaranjitghose** [comment]
Please incorporate the recent changes to the Hackathon section so that the merge conflicts are resolved

**Comment by Atif0604** [comment]
> Please incorporate the recent changes to the Hackathon section so that the merge conflicts are resolved
but it was ready to merge😢

**Comment by Atif0604** [comment]
Hey @smaranjitghose now i can't Fetch Upstream It says i order to fetch upstream i need to make a PR

**Comment by Atif0604** [comment]
What Should i Do now @smaranjitghose

**Comment by smaranjitghose** [comment]
Use the cli.

Check the contribution guidelines

**Comment by Atif0604** [comment]
Hey @smaranjitghose 
Resolved the Conflicts 
Now its good to go

**Review by smaranjitghose — APPROVED** [review]
